File: Arrays/Java/Wave_Array.java
class Solution {
    public static void convertToWave(int n, int[] a) {
        // code here
        Arrays.sort(a);

        for(int i=0; i<n-1; i=i+2) {
            int temp=a[i];
            a[i] = a[i+1];
            a[i+1] = temp;
        }
    }
}

File: Arrays/Python/Wave_Array.py
from typing import List
class Solution:
    def convertToWave(self, n : int, a : List[int]) -> None:
        # code here
        for i in range(0, n-1, 2):
            [a[i], a[i + 1]] = [a [ i + 1], a[i]];
